### PR TITLE
Add CNCF logo to Istio readme

### PR DESCRIPTION
**Pull request by craigbox**
Adds the CNCF logo to the Istio readme as recommended by the project onboarding guidelines.

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -108,3 +108,10 @@ think the issue should get addressed.
 - **Priority**. Each issue has a priority which is represented by the column in the [Prioritization](https://github.com/orgs/istio/projects/6) project. Priority can be one of
 P0, P1, P2, or >P2. The priority indicates how important it is to address the issue within the milestone. P0 says that the
 milestone cannot be considered achieved if the issue isn't resolved.
+
```
Comment (ericvn) [inline README.md:111]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -108,3 +108,10 @@ think the issue should get addressed.
 - **Priority**. Each issue has a priority which is represented by the column in the [Prioritization](https://github.com/orgs/istio/projects/6) project. Priority can be one of
 P0, P1, P2, or >P2. The priority indicates how important it is to address the issue within the milestone. P0 says that the
 milestone cannot be considered achieved if the issue isn't resolved.
+ 
+---
```
Comment (ericvn) [inline README.md:112]:
```suggestion
---
```

**Review by ericvn — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (craigbox) [reply README.md:111]: Were you trying to remove the blank line or spaces in it?  (I think spaces in a blank line are meant to have meaning in Markdown, but I don't think GHFM cares.)

File: README.md [hunk 3]
```diff
@@ -108,3 +108,10 @@ think the issue should get addressed.
...
+
+<div align="center">
+    <img src="https://raw.githubusercontent.com/cncf/artwork/master/other/cncf/horizontal/color/cncf-color.svg" width="300" alt="Cloud Native Computing Foundation logo"/>
+    <p>Istio is a <a href="https://cncf.io">Cloud Native Computing Foundation</a> project.</p>
```
Comment (hzxuzhonghu) [inline README.md:116]: To mention istio is a incubating project?

**Pull request merged**